src/models/Product.js

```diff
@@ -0,0 +1,7 @@
+class Product {
+    constructor() {
+        console.log("Product module");
+    }
+}
+
+module.exports = Product;
```

Comment:
it's common approach, but as using ES6+ standard consider using JS imports/exports. Babel will then transpile all code to `requires/module.exports`
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/import
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/export